docs/api/metric.yaml

```diff
@@ -30,15 +34,17 @@ servers:
 paths:
   /_ping:
     $ref: ./paths/common/_ping.yaml
-  /_metrics:
-    $ref: ./paths/metric/metrics.yaml
```

Comment:
Isn't this what `x-internal` was for? I'm not sure how I feel about the API specs becoming only partially documented based on internal/external opinions again, rather than a comprehensive spec of the API SG is actually serving.

Reply:
Good catch.